src/exo/LoopIR.py

```diff
@@ -163,9 +166,12 @@ def __new__(cls, op):
                 fnarg*          args,
                 expr*           preds,
                 stmt*           body,
-                string?         instr,
+                instr?          instr,
                 srcinfo         srcinfo )
 
+    instr   = ( string          instr,
```

Comment:
Writing `instr.instr` is a little confusing, how about `instr.c_instr` or something like that?

Reply:
Yeah that's better. I will also rename `global_` to `c_global`.

Reply:
Done